src/js/binder.js

```diff
@@ -0,0 +1,128 @@
+/*
+
+    Add persistent bindings between a selector and a model value.  Changes are propagated between the two. See the
+    documentation for more details:
+
+    https://github.com/GPII/gpii-binder/
+
+    This code was originally written by Antranig Basman <amb26@ponder.org.uk> and with his advice was updated and
+    extended by Tony Atkins <tony@raisingthefloor.org>.
+
+*/
+/* global fluid, jQuery */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.binder");
+
+    /**
+     *
+     * Try to evolve a given string into a typed variable or object.
```

Comment:
I note that there is nothing in the docs to alert the user as to what the semantic is here for these compound values